hrms/patches/v15_0/migrate_shift_assignment_schedule_to_shift_schedule.py

```diff
@@ -0,0 +1,29 @@
+import frappe
+from frappe.utils import unique
+
+from hrms.hr.doctype.shift_schedule.shift_schedule import get_or_insert_shift_schedule
+
+
+def execute():
+	frappe.reload_doc("HR", "doctype", "Shift Assignment")
+
+	fields = ["shift_type", "frequency", "employee", "shift_status", "enabled", "create_shifts_after"]
+
+	for doc_name in frappe.get_all("Shift Assignment Schedule", pluck="name"):
+		doc = frappe.db.get_value("Shift Assignment Schedule", doc_name, fields, as_dict=True)
+		repeat_on_days = unique(frappe.get_all("Assignment Rule Day", {"parent": doc_name}, pluck="day"))
```

Comment:
```suggestion
		repeat_on_days = frappe.get_all("Assignment Rule Day", {"parent": doc_name}, pluck="day", distinct=True)
```

Uses db distinct instead of separately iterating over the list and finding unique values